src/test/java/edu/cornell/kfs/tax/fixture/TaxOutputSection.java

```diff
@@ -0,0 +1,37 @@
+package edu.cornell.kfs.tax.fixture;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import edu.cornell.kfs.sys.util.FixtureUtils;
+import edu.cornell.kfs.tax.batch.xml.TaxOutputSectionV2;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.FIELD)
+public @interface TaxOutputSection {
```

Comment:
Do you think it would be appropriate to rename this section TaxOutputSectionFixture?